bower.json

```diff
@@ -20,6 +20,14 @@
     "isotope": "^3.0.4"
   },
   "overrides": {
+    "lazysizes": {
+      "main": [
+        "plugins/bgset/ls.bgset.min.js",
```

Comment:
It does, plugins should go first. as noted [here](https://github.com/aFarkas/lazysizes/#plugins)